privacyidea/lib/utils/__init__.py

```diff
@@ -1135,6 +1139,51 @@ def truncate_comma_list(data, max_len):
     return ",".join(data)
 
 
+def generate_charlists_from_pin_policy(policy):
+    """
+    This function uses the pin content policy string to create the character lists
```

Comment:
Please add an example, so that it is clear, that this is the policy action.

   ....policy string (like "+cns") ...

Reply:
done